e2e/cypress/integration/enterprise/auth_sso/mfa_authentication_spec.js

```diff
@@ -50,45 +50,28 @@ describe('Authentication', () => {
         cy.findByTestId('ServiceSettings.EnforceMultifactorAuthenticationtrue').check();
 
         // # Click "Save".
-        cy.findByText('Save').scrollIntoView().click();
+        cy.findByText('Save').click().wait(TIMEOUTS.ONE_SEC);
```

Comment:
Here is the main fix which adds wait time before checking the URL (see `cy.uiGetMFASecret`). It would be nice to check other UI section/text instead of waiting but there's another case where it's not redirecting to `<site_url>/mfa/setup`.